src/inverter.rs

```diff
@@ -38,7 +38,7 @@ impl<'a> Inverter<'a> {
             info!("Inverter is {new_state:?}");
         }
     }
-
+    
```

Comment:
please run `cargo fmt` to ensure proper formatting and removal of stray whitespace